src/main/update.ts

```diff
@@ -122,6 +132,37 @@ export async function update(update: Update, listeners: IUpdateOptions) {
     return;
   }
 
+  const compatVersion = BigInt(
+    (extra.get("CompatiblityVersion") as string | number) ?? 0
+  );
+  const currentCompatVersion = BigInt(
+    (decodeLocalAPV()?.get("CompatiblityVersion") as string | number) ?? 0
+  );
+
+  if (compatVersion > currentCompatVersion) {
+    console.log(
+      `Stop update process. CompatiblityVersion is higher than current.`
+    );
+    win?.webContents.send("compatiblity-version-higher-than-current");
+    if (win) {
+      const { checkboxChecked } = await dialog.showMessageBox(win, {
+        type: "error",
+        message: t(
+          "This update needs reinstallation due to techincal issues.",
```

Comment:
rather than "This update needs reinstallation due to technical issues", "This version of launcher is no longer supported and requires force re-installation due to technical issues." sounds more coercing.